client/src/components/target/target.js

```diff
@@ -1,195 +1,251 @@
 import React from "react";
-import target from "./target.png";
 
 export default class Target extends React.Component {
   constructor(props) {
     super(props);
-    this.setCanvasRef = element => {
-      this.canvas = element;
-    };
+    this.canvasRef = React.createRef();
```

Comment:
This is a newer way to get ref for an element rendered by React.
this.canvasRef.current accesses the Node and works after rerenders